.github/workflows/gmc-helm.yaml

```diff
@@ -8,13 +8,13 @@ on:
     branches: [main]
     types: [opened, reopened, ready_for_review, synchronize] # added `ready_for_review` since draft is skipped
     paths:
-      - manifests/common/**
```

Comment:
why deleting manifests/common/** from the trigger paths?

Reply:
I don't think helm deployment methods will be different if there is any changes in manifest/common. We requires user to copy the manifest/common directory before doing 'helm install' now.

Reply:
Will manifest/common changes cause failure of GMC helm deployment ?

Reply:
nope, it won't affect this helm chart, which is designed to install GMC/manifests itself, no matter what common manifest files is, the helm chart will only treat them as a bunch of raw files and install them as into the configmap.

However, there could be errors when you launch a GMC pipeline using those manifest files. But that could be catch by `GMC go e2e test`